BadApi/Models/Account/AccountModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BadApi.Helpers;
+using BadApi.Models.Post;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace BadApi.Models.Account
+{
+	public class AccountModel : Domain.Account
```

Comment:
Models should not inherit domain objects - models are for views and domain objects are for data modelling